### DISCUSSION TITLE
Scientific Publishing

### DISCUSSION THREAD

**Discussion by joshhilton** [quarto-dev/quarto-cli#414] [Feature Requests]
Is there any plan to incorporate tools for scientific publishing, such as journal templates for LaTeX exporting?

**Comment by dragonstyle** [comment]
Yes, we would very much like to provide the ability to use templates to target specific journal formats (similar to https://github.com/rstudio/rticles) and provide the ability to create your own templates... We should be working on this soon!

**Comment by cderv** [comment]
Related to this opened issue that you can follow I believe to say achievement of this: https://github.com/quarto-dev/quarto-cli/issues/170

**Comment by mcanouil** [comment]
Closing as completed, see <https://quarto.org/docs/extensions/>.
